### PR TITLE
warn if TV weight is large

### PR DESCRIPTION
**Pull request by tjlane**
This PR addresses #80.

### CODE REVIEW

**Comment by tjlane** [comment]
We need to merge #82 first.

File: meteor/settings.py [hunk 1]
```diff
@@ -45,6 +45,7 @@
 # tv denoising
 TV_WEIGHT_DEFAULT: float = 0.01
 BRACKET_FOR_GOLDEN_OPTIMIZATION: tuple[float, float] = (0.0, 0.05)
+TV_MAX_WEIGHT_EXPECTED = 1.0
```
Comment (alisiafadini) [inline meteor/settings.py:48]: What is the rationale behind choosing 1.0? Empirical observation? `lambda=1.0` means that the fidelity term in the Chambolle equation is divided by 2 exactly, but to me that does not feel like a necessary breaking point... Not saying it's necessarily bad to pick it as a threshold to signal that TV is very high, just pausing/curious

**Review by alisiafadini — APPROVED** [review]

File: meteor/settings.py (hunk 1 above)
Comment (tjlane) [reply meteor/settings.py]: as discussed in person: totally empirical at this point

**Review by alisiafadini — APPROVED** [review]
Yup! Might have to adjust again further down the line but I think these values are our best guesses for defaults rn

**Pull request merged**